pkgs/top-level/all-packages.nix

```
@@ -28861,6 +28861,17 @@ with pkgs;
 
   picom = callPackage ../applications/window-managers/picom {};
 
+  picom-next = picom.overrideAttrs (oldAttrs: rec {
+    version = "vNext";
+      src = fetchFromGitHub {
+      owner = "yshui";
+      repo = "picom";
+      rev = "fade045eadf171d2c732820d6ebde7d1943a1397";
+      sha256 = "fPiLZ63+Bw5VCxVNqj9i5had2YLa+jFMMf85MYdqvHU=";
+  };
+    maintainers = with maintainers; oldAttrs.maintainers ++ [ GKasparov ];
```

Sorry :disappointed:.

```suggestion
    meta.maintainers = with maintainers; oldAttrs.maintainers ++ [ GKasparov ];
```

---

Nws, my fault